src/java/main/org/apache/zookeeper/server/ContainerManager.java

```diff
@@ -116,8 +116,8 @@ public void checkContainers()
             Request request = new Request(null, 0, 0,
                     ZooDefs.OpCode.deleteContainer, path, null);
             try {
-                LOG.info("Attempting to delete candidate container: %s",
-                        containerPath);
+                LOG.info(String.format("Attempting to delete candidate container: %s",
```

Comment:
As this a log message you could use 
`LOG.info("Attempting to delete candidate container: {}", container Path);`

Reply:
to use the same logging style as the other code.

                LOG.error(String.format("Could not delete container: %s" ,
                        containerPath), e);

Reply:
+1 for @eribeiro 
We already use that form at various places in the codebase, so feel free to use the better one. Existing String.formats should be refactored in the long term, but that probably will happen in small refactorings like this.